roboteam_ai/src/skills/GoToPosLuTh.cpp

```diff
@@ -33,8 +33,10 @@ GoToPosLuTh::Status GoToPosLuTh::onUpdate() {
     displayData.clear();
 
     if (goToBall) {
-        auto ball = World::getBall();
-        targetPos = ball.pos;
+        if (!ball) {
+            ROS_ERROR("GoToPosLuTh: No ball found! assuming (%f, %f)", ball->pos.x, ball->pos.y);
```

Comment:
ball is a null pointer in this case, so this will segfault as you try to access it in the ros error.